<ui version="4.0" >
 <class>TwoKnobsBase</class>
 <widget class="QWidget" name="TwoKnobsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>TwoKnobs</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QwtKnob" name="_knob1" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLCDNumber" name="_lcd1" >
         <property name="minimumSize" >
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_label1" >
         <property name="minimumSize" >
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>title</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QwtKnob" name="_knob2" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLCDNumber" name="_lcd2" >
         <property name="minimumSize" >
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_label2" >
         <property name="minimumSize" >
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>title</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_knob1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>_lcd1</receiver>
   <slot>display(double)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>45</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel" >
     <x>55</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_knob2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>_lcd2</receiver>
   <slot>display(double)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>69</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel" >
     <x>63</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
